src/Compilers/VisualBasic/Portable/Symbols/Source/LocalSymbol.vb

```diff
@@ -664,31 +664,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Friend Overrides Function ComputeTypeInternal(localBinder As Binder) As TypeSymbol
 
                 Dim diagBag = DiagnosticBag.GetInstance()
-
-                Dim collectionExpression As BoundExpression = Nothing
-                Dim elementType As TypeSymbol = Nothing
-                Dim isEnumerable = False
-                Dim boundGetEnumeratorCall As BoundExpression = Nothing
-                Dim boundEnumeratorPlaceholder As BoundLValuePlaceholder = Nothing
-                Dim boundMoveNextCall As BoundExpression = Nothing
-                Dim boundCurrentCall As BoundExpression = Nothing
-                Dim collectionPlaceholder As BoundRValuePlaceholder = Nothing
-                Dim needToDispose = False
-                Dim isOrInheritsFromOrImplementsIDisposable = False
                 Dim type As TypeSymbol = Nothing
 
                 type = localBinder.InferForEachVariableType(Me,
                                                        _collectionExpressionSyntax,
-                                                       collectionExpression,
-                                                       elementType,
-                                                       isEnumerable,
-                                                       boundGetEnumeratorCall,
-                                                       boundEnumeratorPlaceholder,
-                                                       boundMoveNextCall,
-                                                       boundCurrentCall,
-                                                       collectionPlaceholder,
-                                                       needToDispose,
-                                                       isOrInheritsFromOrImplementsIDisposable,
+                                                       collectionExpression:=Nothing,
```

Comment:
I don't see any functional changes in this file? Consider reverting changes.

Reply:
> I don't see any functional changes in this file? Consider reverting changes.

There are functional changes here.

---
In reply to: [181876878](https://github.com/dotnet/roslyn/pull/26158#discussion_r181876878) [](ancestors = 181876878)